Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CentralRepoDbChoice.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Central Repository
+ *
+ * Copyright 2015-2020 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.centralrepository.datamodel;
+
+/**
+ * the database choices available for central repo
+ */
+public class CentralRepoDbChoice {
```

Comment:
Could this be an enum? Then we could use case statements later.

Reply:
nevermind.  I got over my mental block.  I think this is a lot more ergonomic.